### ISSUE TITLE
Can't locate Path/Class.pm in @INC

### ISSUE THREAD

**Issue by eserte** [hoelzro/dist-zilla-plugin-git-excludeuntracked#4]
The test suite fails on some on my smoker, possibly because of an undeclared dependency:
```
->file called on a Dist::Zilla::Path object; this will cease to work in Dist::Zilla v7; downstream code should be updated to use Path::Tiny API, not Path::Class at t/01-basic.t line 28.
Can't locate Path/Class.pm in @INC (you may need to install the Path::Class module) (@INC contains: ...  .) at /home/cpansand/.cpan/build/2020010903/Dist-Zilla-6.012-0/blib/lib/Dist/Zilla/Path.pm line 40.
# Looks like your test exited with 2 before it could output anything.
t/01-basic.t ............. 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 1/1 subtests 
->file called on a Dist::Zilla::Path object; this will cease to work in Dist::Zilla v7; downstream code should be updated to use Path::Tiny API, not Path::Class at t/02-git-subdirectory.t line 36.
Can't locate Path/Class.pm in @INC (you may need to install the Path::Class module) (@INC contains: ...  .) at /home/cpansand/.cpan/build/2020010903/Dist-Zilla-6.012-0/blib/lib/Dist/Zilla/Path.pm line 40.
# Looks like your test exited with 2 before it could output anything.
t/02-git-subdirectory.t .. 
Dubious, test returned 2 (wstat 512, 0x200)
Failed 1/1 subtests 
```

**Comment by hoelzro** [comment]
@eserte Thanks for the report! I've deprecated this module; is there a way to set it up to be removed from your smokers?

**Comment by eserte** [comment]
There seem to be modules depending on your module, see
https://metacpan.org/requires/distribution/Dist-Zilla-Plugin-Git-ExcludeUntracked?sort=[[2,1]]&size=100
So even if I wouldn't directly smoke your module anymore, it could get built because of one of the revdeps.

**Comment by hoelzro** [comment]
@eserte That's a good point - I just released version 0.06 with a fix.  Could you verify the fix and let me know?

**Comment by eserte** [comment]
It looks better now: http://fast-matrix.cpantesters.org/?dist=Dist-Zilla-Plugin-Git-ExcludeUntracked%200.06
